BladeSDK/src/main/java/io/bladewallet/bladesdk/Utils.kt

```diff
@@ -17,10 +17,15 @@ import com.google.gson.Gson
 
 
 internal suspend fun getRemoteConfig(network: String, dAppCode: String, sdkVersion: String, bladeEnv: BladeEnv): RemoteConfig = withContext(Dispatchers.IO) {
-    val url: String = if (bladeEnv === BladeEnv.Prod) {
-        "https://rest.prod.bladewallet.io/openapi/v7/sdk/config"
+    val url: String
+    val fallbackConfig = RemoteConfig(fpApiKey = "")
+
+    if (bladeEnv === BladeEnv.Prod) {
+        url = "https://rest.prod.bladewallet.io/openapi/v7/sdk/config"
+        fallbackConfig.fpApiKey = "Li4RsMbgPldpOVfWjnaF"
```

Comment:
why we are exposing it in sources?

Reply:
in case of failing get /sdk/config returning public fingerprint api key to still init sdk. this behaviour copied from swift-blade